### PR TITLE
Fix: Cash Bounty Logic

### PR DESCRIPTION
**Pull request by commy2**
- Cash Bounty no longer handled by any object
- fixes:

- restores Cash Bounty Command Center scaffhold logic removed in https://github.com/TheSuperHackers/GeneralsGamePatch/pull/595
- spawns an invisible, immortal Cash Bounty handling dummy on skirmish and multiplayer maps if faction is GLAx or Boss, logic restored that was removed in: https://github.com/TheSuperHackers/GeneralsGamePatch/pull/2263

Cash Bounty now requires:

- On Skirmish / MP / GC: The genpower, nothing else
- In campaign (only relevant map is GLA05) / custom spotless maps: The genpower + a GLA/Boss Command Center (scaffold), same as 1.04

### CODE REVIEW

File: Patch104pZH/GameFilesEdited/Data/INI/Object/System.ini [hunk 1]
```diff
@@ -3084,6 +3084,52 @@ Object DummyWeaponProjectile
 
 End
 
+; Patch104p @bugfix commy2 24/10/2021 Logic that grants player Cash Bounty.
+; Patch104p @bugfix commy2 15/08/2022 Do not use HighlanderBody for these dummies: It will cause unexpected resistance to particle beams.
+; Patch104p @bugfix commy2 20/08/2022 Use DeletionUpdate for ImmortalBody, because immortals cannot die.
+;------------------------------------------------------------------------------
+Object CashBountyDummy
+
+
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/System.ini]: Excess white space in Object.

**Review by xezon — APPROVED** [review]

**Comment by MTKing4** [comment]
Why is this merged?
We have 2 people ok with the change and 2 not. Meaning it's undecided yet.
Me and @ImTimK stated that a CC requirement is the better approach. This buffs GLAs and gives less incentives to build a $2000 CC at rank 3 for cash bounty

**Comment by commy2** [comment]
Your comment is on the wrong PR. This a bugfix follow up.

**Comment by MTKing4** [comment]
Isn't this bug fix only needed for this approach? I already commented on the main report

**Comment by commy2** [comment]
Yes, hence "follow up".

**Pull request merged**